modules/ROOT/pages/anypoint-monitoring.adoc

```diff
@@ -0,0 +1,182 @@
+= Anypoint Monitoring on Anypoint Platform PCE
+
+Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) includes support for Anypoint Monitoring.
```

Comment:
We need to define what a unit of "Monitoring" is. Here's an idea for the first paragraph in this page:

> Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) includes support for Anypoint Monitoring. Anypoint Monitoring is a suite of tools that provides feedback from Mule flows and components in your application network.
> 
> Implementation of Anypoint Monitoring for PCE requires the "Monitoring for On-Premises Management" subscription. This subscription entitles you to two Monitoring Instances. A Monitoring Instance is an implementation of the 3-node Monitoring configuration as described in PCE Hardware Prerequisites. Each Monitoring Instance is dedicated to a single PCE Instance. For customers who whish to implement Anypoint Monitoring with more than 2 PCE Instances, you must purchase additional Monitoring Instances. Please contact your customer success representative for more information.

Reply:
See Slack conversation with Lisa.